Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -332,23 +333,29 @@ namespace uWS {
             memcpy(dst + headerLength, src, length);
 
             if ( !isServer ) {
-                uint32_t mask_i32 = *reinterpret_cast<uint32_t*>(mask);
+                std::byte* start_byte = dst + headerLength;
+                std::byte* end_byte   = start_byte + length;
+                size_t     i          = 0;
 
-                auto*     start = reinterpret_cast<uint32_t*>(dst + headerLength);
-                uint32_t* end   = start + length / 4;
+                // Handle first x amount of bytes individually until we are aligned with the 4-byte alignment
+                for ( ; reinterpret_cast<uintptr_t>(start_byte) % 4 != 0 && start_byte != end_byte; ++i ) {
```

Comment:
Small improvement here:  The standard way to check for alignment rather than `% 4` is `& 3`.